Development/src/components/ActiveField.js

```diff
@@ -60,10 +60,11 @@ const ActiveField = ({ className, source, record = {}, resource, ...rest }) => {
 
     const handleChange = (record, resource) => {
         toggleMasterEnable(record, resource)
-            .then(() => setChecked(!checked))
+            .then(({ data }) => setChecked(get(data, 'master_enable')))
```

Comment:
So `toggleMasterEnable` returns the new IS-05 /active endpoint data does it?

Reply:
Yes, thought it best to base the status of the switch off the response from the IS-05 endpoint rather than simply toggling it